src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java

```diff
@@ -189,13 +190,21 @@ public void setPresenter(Presenter presenter) {
 	@Override
 	public void showMemberMenuItems() {
 		leaveTeamItem.setVisible(true);
+		// TODO: remove next line to take out of alpha mode
+		teamProjectsItem.setVisible(DisplayUtils.isInTestWebsite(cookieProvider));
```

Comment:
Must be a team member for this command to work, so only show it when other member items are shown.